wfe/wfe.go

```diff
@@ -1174,6 +1175,43 @@ func (wfe *WebFrontEndImpl) Authorization(ctx context.Context, logEvent *request
 		return
 	}
 
+	if wfe.AllowAuthzDeactivation && request.Method == "POST" {
+		body, _, reg, prob := wfe.verifyPOST(ctx, logEvent, request, true, core.ResourceAuthz)
+		addRequesterHeader(response, logEvent.Requester)
+		if prob != nil {
+			wfe.sendError(response, logEvent, prob, nil)
+			return
+		}
+		if reg.ID != authz.RegistrationID {
+			logEvent.AddError("registration ID doesn't match ID for authorization")
```

Comment:
I think there's no need to do an AddError for an error that exactly matches the problemdetails that we provide to sendError, right?